Drop-In/src/main/java/com/braintreepayments/api/PaymentMethodNonceInspector.java

```diff
@@ -0,0 +1,32 @@
+package com.braintreepayments.api;
+
+class PaymentMethodNonceInspector {
```

Comment:
You've _cast_ a warm feeling on my heart with this one.  It was such an annoying thing to do.

Reply:
:nod: The `DropInPaymentMethodType` class was doing a lot of work. We didn't get heavy into the value objects yet but I have a feeling we can clean up more of the enum code.